database/src/pool/postgres.rs

```diff
@@ -361,6 +364,19 @@ static MIGRATIONS: &[&str] = &[
     );
     CREATE INDEX IF NOT EXISTS job_queue_request_tag_idx ON job_queue (request_tag);
     "#,
+    // The name is unique and not null in the collector config and is simpler
+    // to use as we do not expose the `collector_id` in the code
+    r#"
+    ALTER TABLE job_queue DROP CONSTRAINT IF EXISTS job_queue_collector;
+    ALTER TABLE job_queue ADD COLUMN IF NOT EXISTS collector_name TEXT;
```

Comment:
Why did you add the `IF NOT EXISTS` clause? That shouldn't be needed, right?

Reply:
52bfe1f796c47d852b54a6c7d76f64eaa099740a